src/core/admittance_matrix.jl

```diff
@@ -47,7 +47,7 @@ function calc_admittance_matrix(data::Dict{String,<:Any})
         if branch[pm_component_status["branch"]] != pm_component_status_inactive["branch"] && haskey(bus_to_idx, f_bus) && haskey(bus_to_idx, t_bus)
             f_bus = bus_to_idx[f_bus]
             t_bus = bus_to_idx[t_bus]
-            y = inv(branch["br_r"] + branch["br_x"]im)
+            y = 1 / (branch["br_r"] + branch["br_x"]im)
             tr, ti = calc_branch_t(branch)
```

Comment:
At some point I was told `inv` is more numerically stable for complex numbers than `/` (I think from @frederikgeth ?), so we might restore this to the original, if we don't have a reason to prefer `/`.

Reply:
Yeah I'm tempted to just close this PR. I don't understand the Windows failure, and I don't have a local machine to debug

Reply:
But now we know the underlying source of the numerical issue. And the Windows problem looks like it has a hard time with primal feasibility, which might make sense if the columns of the susceptance matrix are not linearly dependent?

Reply:
I am find to close this PR, but I would like to get the `eigvals` update into master, if you think it is better than the `det` approach.